applications/Unity.GrantManager/modules/Unity.Payments/src/Unity.Payments.Application/Domain/PaymentRequests/IPaymentRequestRepository.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Unity.Payments.Domain.PaymentRequests
 {
-    public interface IPaymentRequestRepository : IBasicRepository<PaymentRequest, Guid>
-    {
+    public interface IPaymentRequestRepository : IRepository<PaymentRequest, Guid>
```

Comment:
Please don't use the IRepository, we used it in the main app but we shouldn't trying to implement the modules as per best practices.
https://docs.abp.io/en/abp/latest/Best-Practices/Repositories